src/libstore/store-api.hh

```diff
@@ -445,7 +448,7 @@ public:
        sizes) of a set of paths.  If a path does not have substitute
        info, it's omitted from the resulting ‘infos’ map. */
     virtual void querySubstitutablePathInfos(const StorePathSet & paths,
-        SubstitutablePathInfos & infos) { return; };
+        SubstitutablePathInfos & infos, std::map<std::string, Derivation> drvs = {}) { return; };
```

Comment:
Ought to be `std::map<std::string /* StorePath */, std::map<std::string /* output name */, std::string /* ContentAddress */>>`

Reply:
For back-compat, we can overload the path, and turn the set into a map with all empty strings.